<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="14" tests="39" time="2553102">
 <testsuite name="Page load" timestamp="2018-08-16T15:04:59" hostname="localhost" time="0" errors="0" tests="0" skipped="0" disabled="0" failures="0">
 </testsuite>
 <testsuite name="Login Page" timestamp="2018-08-16T15:04:59" hostname="localhost" time="40.541" errors="0" tests="5" skipped="0" disabled="0" failures="3">
  <testcase classname="Login Page" name="should display incorrect userid error message when entering incorrect userid" time="11.037" />
  <testcase classname="Login Page" name="should display required field error message and not go to main menu when password entering as blank" time="9.799" />
  <testcase classname="Login Page" name="should display required field error message and not go to main menu when username entering as blank" time="9.627" />
  <testcase classname="Login Page" name="should display incorrect password error message when entering incorrect password" time="9.72">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at UserContext.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:37:37)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7]]>
   </failure>
   <failure type="exception" message="Failed: No element found using locator: By(xpath, //*[@id=&quot;AuthSignIn&quot;]/form/md-dialog-content/div/div[2]/div[3]/span/span)"><![CDATA[NoSuchElementError: No element found using locator: By(xpath, //*[@id="AuthSignIn"]/form/md-dialog-content/div/div[2]/div[3]/span/span)
    at elementArrayFinder.getWebElements.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as isDisplayed] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as isDisplayed] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at UserContext.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:38:24)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
From: Task: Run it("should display incorrect password error message when entering incorrect password") in control flow
    at UserContext.<anonymous> (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:33:2)
    at addSpecsToSuite (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:6:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)]]>
   </failure>
  </testcase>
  <testcase classname="Login Page" name="should go to main menu when entering correct password" time="0.357">
   <failure type="exception" message="Failed: No element found using locator: By(css selector, *[id=&quot;username&quot;])"><![CDATA[NoSuchElementError: No element found using locator: By(css selector, *[id="username"])
    at elementArrayFinder.getWebElements.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as clear] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as clear] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at Object.login (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.po.js:10:12)
    at UserContext.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:43:7)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it("should go to main menu when entering correct password") in control flow
    at UserContext.<anonymous> (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:42:3)
    at addSpecsToSuite (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\Login\LoginPage.spec.js:6:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)]]>
   </failure>
  </testcase>
 </testsuite>
 <testsuite name="Landing Page" timestamp="2018-08-16T15:05:40" hostname="localhost" time="141.243" errors="0" tests="7" skipped="0" disabled="0" failures="7">
  <testcase classname="Landing Page" name="should have Option “About Pharmacuity” when login with any valid user" time="21.659">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:29:53
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
  <testcase classname="Landing Page" name="should have Option “Metrics and Benchmarking” when login with any valid user" time="21.586">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:45:48
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
  <testcase classname="Landing Page" name="should have Option “Site,PI &amp; Recuitment Optimization” when login with any valid user" time="19.557">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:62:48
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
  <testcase classname="Landing Page" name="should have Option “Inovvative Protocol Design” when login with any valid user" time="19.382">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:79:48
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
  <testcase classname="Landing Page" name="should have Option “Trial Forcasting” when login with any valid user" time="19.855">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:95:48
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
  <testcase classname="Landing Page" name="the user should be able to logout and not able to access menu untill login again" time="22.579" />
  <testcase classname="Landing Page" name="should display only 2 options “About Pharmacuity” and “Metrics and Benchmarking” when login with the Roche user" time="16.621">
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:124:48
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
   <failure type="toBe" message="Expected false to be true."><![CDATA[Error: Failed expectation
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\landingPage\LandingPage.spec.js:128:50
    at elementArrayFinder_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
 </testsuite>
 <testsuite name="Metrics And Benchmarking Page" timestamp="2018-08-16T15:08:01" hostname="localhost" time="429.914" errors="0" tests="15" skipped="0" disabled="0" failures="0">
  <testcase classname="Metrics And Benchmarking Page" name="should not display filter option" time="26.678" />
  <testcase classname="Metrics And Benchmarking Page" name="should display Dashboard option" time="28.41" />
  <testcase classname="Metrics And Benchmarking Page" name="should display PF-SA option" time="27.933" />
  <testcase classname="Metrics And Benchmarking Page" name="should display SA-FPI option" time="28.637" />
  <testcase classname="Metrics And Benchmarking Page" name="should display FPI-LPI option" time="29.101" />
  <testcase classname="Metrics And Benchmarking Page" name="should display PF-10THPI option" time="28.962" />
  <testcase classname="Metrics And Benchmarking Page" name="should display ERATE option" time="28.866" />
  <testcase classname="Metrics And Benchmarking Page" name="should display E BINS option" time="29.041" />
  <testcase classname="Metrics And Benchmarking Page" name="should display QUADA option" time="28.979" />
  <testcase classname="Metrics And Benchmarking Page" name="should display #S-100P option" time="28.685" />
  <testcase classname="Metrics And Benchmarking Page" name="should display P(P) option" time="29.198" />
  <testcase classname="Metrics And Benchmarking Page" name="should display P(C) option" time="28.644" />
  <testcase classname="Metrics And Benchmarking Page" name="should display P/SITE option" time="29.027" />
  <testcase classname="Metrics And Benchmarking Page" name="should display Export to excel option" time="28.595" />
  <testcase classname="Metrics And Benchmarking Page" name="should display Help option" time="29.155" />
 </testsuite>
 <testsuite name="M&amp;B PF-SA Page" timestamp="2018-08-16T15:15:11" hostname="localhost" time="805.484" errors="0" tests="6" skipped="0" disabled="0" failures="3">
  <testcase classname="M&amp;B PF-SA Page" name="should be matched Studies Box Plots value of Sponsor with the database value for the given filter options" time="179.808" />
  <testcase classname="M&amp;B PF-SA Page" name="should be matched Average Box Plots value of Sponsor with the database value for the given filter options" time="166.832" />
  <testcase classname="M&amp;B PF-SA Page" name="should be matched Median Box Plots value of Sponsor with the database value for the given filter options" time="167.185" />
  <testcase classname="M&amp;B PF-SA Page" name="should be matched Studies Box Plots value of Competition with the database value for the given filter options" time="177.14" />
  <testcase classname="M&amp;B PF-SA Page" name="should be matched Average Box Plots value of Competition with the database value for the given filter options" time="85.199">
   <failure type="exception" message="Failed: unknown error: Element &lt;div ng-repeat=&quot;i in filterComponents&quot; ng-if=&quot;i.filterCategory==&apos;common&apos;&quot; class=&quot;ng-scope&quot;&gt;...&lt;/div&gt; is not clickable at point (121, 136). Other element would receive the click: &lt;div class=&quot;md-dialog-container ng-scope&quot; tabindex=&quot;-1&quot; style=&quot;top: 0px; height: 618px;&quot;&gt;...&lt;/div&gt;
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)"><![CDATA[WebDriverError: unknown error: Element <div ng-repeat="i in filterComponents" ng-if="i.filterCategory=='common'" class="ng-scope">...</div> is not clickable at point (121, 136). Other element would receive the click: <div class="md-dialog-container ng-scope" tabindex="-1" style="top: 0px; height: 618px;">...</div>
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)
    at Object.checkLegacyResponse (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: WebElement.click()
    at thenableWebDriverProxy.schedule (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at WebElement.schedule_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
    at WebElement.click (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2092:17)
    at actionFn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:89:44)
    at Array.map (<anonymous>)
    at actionResults.getWebElements.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:461:65)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)Error
    at ElementArrayFinder.applyAction_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at element.all.filter.each (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:1274:23)
    at arr.map (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:628:33)
    at Array.map (<anonymous>)
    at asElementFinders_.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:627:28)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
From: Task: Run it("should be matched Average Box Plots value of Competition with the database value for the given filter options") in control flow
    at UserContext.<anonymous> (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:64:48
    at ControlFlow.emit (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
    at ControlFlow.shutdown_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
    at shutdownTask_.MicroTask (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2599:53)
From asynchronous test: 
Error
    at Suite.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:1237:3)
    at addSpecsToSuite (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:28:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)]]>
   </failure>
  </testcase>
  <testcase classname="M&amp;B PF-SA Page" name="should be matched Median Box Plots value of Competition with the database value for the given filter options" time="29.319">
   <failure type="exception" message="Failed: unknown error: Element &lt;md-menu-item ng-repeat=&quot;phx in ctrl.phxModules&quot; ng-click=&quot;ctrl.changeSelectedApp($index)&quot; ng-show=&quot;!ctrl._isDisabled(&apos;PharmAcuity&apos;)&quot; class=&quot;ng-scope&quot; role=&quot;button&quot; tabindex=&quot;0&quot; aria-hidden=&quot;false&quot; style=&quot;color: Red; border: 2px solid red;&quot;&gt;...&lt;/md-menu-item&gt; is not clickable at point (423, 57). Other element would receive the click: &lt;md-backdrop class=&quot;md-menu-backdrop md-click-catcher ng-scope&quot; style=&quot;height: 618px;&quot;&gt;&lt;/md-backdrop&gt;
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)"><![CDATA[WebDriverError: unknown error: Element <md-menu-item ng-repeat="phx in ctrl.phxModules" ng-click="ctrl.changeSelectedApp($index)" ng-show="!ctrl._isDisabled('PharmAcuity')" class="ng-scope" role="button" tabindex="0" aria-hidden="false" style="color: Red; border: 2px solid red;">...</md-menu-item> is not clickable at point (423, 57). Other element would receive the click: <md-backdrop class="md-menu-backdrop md-click-catcher ng-scope" style="height: 618px;"></md-backdrop>
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)
    at Object.checkLegacyResponse (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: WebElement.click()
    at thenableWebDriverProxy.schedule (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at WebElement.schedule_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
    at WebElement.click (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2092:17)
    at actionFn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:89:44)
    at Array.map (<anonymous>)
    at actionResults.getWebElements.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:461:65)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)Error
    at ElementArrayFinder.applyAction_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\MetricsAndBenchmarking\MetricsAndBenchmarking.po.js:12:24
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
From: Task: Run beforeEach in control flow
    at UserContext.<anonymous> (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:30:3)
    at addSpecsToSuite (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:28:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)]]>
   </failure>
   <failure type="exception" message="Failed: No element found using locator: By(xpath, //*[@id=&quot;content&quot;]/div/div/div/div/div/div[2]/header/div[1]/input)"><![CDATA[NoSuchElementError: No element found using locator: By(xpath, //*[@id="content"]/div/div/div/div/div/div[2]/header/div[1]/input)
    at elementArrayFinder.getWebElements.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as clear] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as clear] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at Object.setSitePercentZero (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.po.js:210:24)
    at UserContext.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:1370:10)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it("should be matched Median Box Plots value of Competition with the database value for the given filter options") in control flow
    at UserContext.<anonymous> (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
    at Function.next.fail (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4274:9)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:64:48
    at ControlFlow.emit (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\events.js:62:21)
    at ControlFlow.shutdown_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2674:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:1367:3)
    at addSpecsToSuite (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\PF_To_SA\PF_To_SA.spec.js:28:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)]]>
   </failure>
  </testcase>
 </testsuite>
 <testsuite name="M&amp;B FPI-LPI Page" timestamp="2018-08-16T15:28:37" hostname="localhost" time="1135.92" errors="0" tests="6" skipped="0" disabled="0" failures="1">
  <testcase classname="M&amp;B FPI-LPI Page" name="should be matched Studies Box Plots value of Sponsor with the database value for the given filter options" time="420.814" />
  <testcase classname="M&amp;B FPI-LPI Page" name="should be matched Average Box Plots value of Sponsor with the database value for the given filter options" time="328.799" />
  <testcase classname="M&amp;B FPI-LPI Page" name="should be matched Median Box Plots value of Sponsor with the database value for the given filter options" time="348.749" />
  <testcase classname="M&amp;B FPI-LPI Page" name="should be matched Studies Box Plots value of Competition with the database value for the given filter options" time="36.603">
   <failure type="exception" message="Failed: unknown error: Element &lt;md-menu-item ng-repeat=&quot;phx in ctrl.phxModules&quot; ng-click=&quot;ctrl.changeSelectedApp($index)&quot; ng-show=&quot;!ctrl._isDisabled(&apos;PharmAcuity&apos;)&quot; class=&quot;ng-scope&quot; role=&quot;button&quot; tabindex=&quot;0&quot; aria-hidden=&quot;false&quot; style=&quot;color: Red; border: 2px solid red;&quot;&gt;...&lt;/md-menu-item&gt; is not clickable at point (423, 57). Other element would receive the click: &lt;md-backdrop class=&quot;md-menu-backdrop md-click-catcher ng-scope&quot;&gt;&lt;/md-backdrop&gt;
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)"><![CDATA[WebDriverError: unknown error: Element <md-menu-item ng-repeat="phx in ctrl.phxModules" ng-click="ctrl.changeSelectedApp($index)" ng-show="!ctrl._isDisabled('PharmAcuity')" class="ng-scope" role="button" tabindex="0" aria-hidden="false" style="color: Red; border: 2px solid red;">...</md-menu-item> is not clickable at point (423, 57). Other element would receive the click: <md-backdrop class="md-menu-backdrop md-click-catcher ng-scope"></md-backdrop>
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.16299 x86_64)
    at Object.checkLegacyResponse (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
From: Task: WebElement.click()
    at thenableWebDriverProxy.schedule (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at WebElement.schedule_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
    at WebElement.click (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2092:17)
    at actionFn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:89:44)
    at Array.map (<anonymous>)
    at actionResults.getWebElements.then (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:461:65)
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)Error
    at ElementArrayFinder.applyAction_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\MetricsAndBenchmarking\MetricsAndBenchmarking.po.js:12:24
    at ManagedPromise.invokeCallback_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at <anonymous>
From: Task: Run it("should be matched Studies Box Plots value of Competition with the database value for the given filter options") in control flow
    at UserContext.<anonymous> (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\M&B-FPI_To_LPI\FPI_To_LPI.spec.js:446:3)
    at addSpecsToSuite (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\DibaS\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (D:\PharmAcuity_AutomationTesting\pharmacuity-ui-automation\M&B-FPI_To_LPI\FPI_To_LPI.spec.js:27:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)]]>
   </failure>
  </testcase>
  <testcase classname="M&amp;B FPI-LPI Page" name="should be matched Average Box Plots value of Competition with the database value for the given filter options" time="0.875" />
  <testcase classname="M&amp;B FPI-LPI Page" name="should be matched Median Box Plots value of Competition with the database value for the given filter options" time="0.063" />
 </testsuite>
</testsuites>